staubli_tx2_90_support/package.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>staubli_tx2_90_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the Staubli TX2-90 (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Staubli TX2-90 manipulators. This includes the base model,
+      as well as models with extended arm and forearm links (L and XL).
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>TX2-90 - Standard Arm - Nominal Speed</li>
+      <li>TX2-90L - Long Arm - Nominal Speed</li>
+      <li>TX2-90XL - Extra Long Arm - Nominal Speed</li>
+    </ul>
+    <p>
+      Joint limits were taken from broschures, torque limits, and maximum joint velocities
+      are based on the
+      information in the <em>Staubli Robotics - TX series 90 family -
+      Instruction Manual</em> version <em>D28082904C – 01/12/2014</em> and might have changed.
```

Comment:
Is there any particular reason the joint limits were not taken from the cited manual?